portfolio/src/main/webapp/projects.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+  <body>
+    <div id="navbar">
+        <ul>
+          <li><a href="index.html">home</a></li> 
+          <li><a href="projects.html">projects</a></li> 
+          <li><a href="blog.html">blog</a></li> 
+          <li><a href="photos.html">photos</a></li> 
+          <li><a href="random.html">random</a></li>
+        </ul>
+    </div> 
```

Comment:
overkill comment for this starter project, but just as an fyi
-- navbar isn't reusable even though i assume you'll be using the same div in all the various pages. so changing the navbar requires changing each of this replica in all the pages.

later for the capstone or more complicated front-end, having some form of templating (either through importing the HTML, or through some framework like React, etc).

again, no need to act on this but just a random note.